x-pack/plugins/observability_solution/infra/public/components/asset_details/charts/container_charts.tsx

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Just an idea: I wonder if we could structure these chart files as follows:

- `kubernetes_charts.tsx` -> contains `KubernetesNodeCharts` (existing renamed) and `KubernetesContainerCharts` (new)
- `docker_charts.tsx` -> contains the new `DockerCharts`
- `host_charts.tsx` -> remains unchaged.

- `metrics/container_metrics.tsx` -> would decide what to render instead of `ContainerCharts in `container_charts.tsx`

I don't know, I find the `kubernetes_charts.tsx` a bit out of place. Do you have any suggestions?

Reply:
Thanks, I like the first proposal, I will do it in this PR already